### PR TITLE
fix(kalert,kemptystate): icon slot test

### PR DESCRIPTION
**Pull request by kaiarrowood**
# Summary

## PR Checklist

* [x] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [x] **Tests coverage:** test coverage was added for new features and bug fixes
* [ ] **Docs:** includes a technically accurate README

### CODE REVIEW

**Review by portikM — APPROVED** [review]
LGTM 🚀

File: src/components/KAlert/KAlert.cy.ts [hunk 1]
```diff
@@ -85,21 +86,25 @@ describe('KAlert', () => {
...
-        default: `<span data-testid="default-slot-content">${defaultSlotContent}</span>`,
+        default: h('span', {}, defaultSlotContent),
       },
     })
 
     cy.get('.alert-message').should('be.visible').and('have.text', defaultSlotContent)
   })
 
   it('displays icon passed through icon slot', () => {
+    const dragon = '🐉'
```
Comment (adamdehaven) [inline src/components/KAlert/KAlert.cy.ts:97]: can we just check for a string here instead of an emoji?

**Review by adamdehaven — CHANGES_REQUESTED** [review]

File: src/components/KAlert/KAlert.cy.ts (hunk 1 above)
Comment (kaiarrowood) [reply src/components/KAlert/KAlert.cy.ts:97]: I can change this, but for the record the test is completely functional as is and this shouldn't be considered blocking feedback ☹️

Comment (adamdehaven) [reply src/components/KAlert/KAlert.cy.ts:97]: We shouldn't be using emojis in tests 🤷🏼

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**